cg/cli/set/case.py

```diff
@@ -77,18 +80,25 @@ def set_case(
     if panel_abbreviations:
         update_panels(case=case, panel_abbreviations=panel_abbreviations, status_db=status_db)
 
-    if priority:
+    if isinstance(priority, Priority):
         update_priority(case=case, priority=priority)
 
     status_db.session.commit()
 
 
-def abort_on_empty_options(options: list[str]) -> None:
-    if not any(options):
+def abort_on_empty_options(
+    options: list[str], bypass_check: Callable[[any], bool], priority: Priority | None
+) -> None:
+    # Abort if options are empty and bypass_check for priority returns False
+    if not any(options) and not bypass_check(priority):
```

Comment:
This check seems unclearer to me than just saying `if len(options) == 0:`. Is there a reason for writing it like this?

Reply:
len option is never smaller than one, since it is a list full of stuff. I tried that initially but it broke loads of tests.

Reply:
I do see, we will have a long list with multiple None entries